### PR TITLE
Development

### PR DESCRIPTION
**Pull request by LukPrasek**
I have added the new collections (ArrayList)....
Added new function...

### CODE REVIEW

File: src/Main.java [hunk 1]
```diff
@@ -78,16 +70,46 @@ private static void userInterface(boolean isCorrectCarData) {
 		userInterface(isCorrectCarData);// REKURENCJA - WYWOLUJE SIE PONOWNIE
 										// METODA
 	}
+
+	private static void addEmployeeM() throws IncorrectUserInterfaceDataException {
```
Comment (LukPrasek) [inline src/Main.java:74]: please, have a look deeper in this section

File: src/car/CarObjectsStorage.java [hunk 2]
```diff
@@ -13,6 +13,14 @@ public CarObjectsStorage() {
 		table[2] = AddCar.CarWithThreeSeats("Mercedes",  "WB0987632", "2015", new Driver(ServiceMethods.findEmployeeByID(5)), new Seat(ServiceMethods.findEmployeeByID(9)), new Seat(ServiceMethods.findEmployeeByID(1)));;
 		table[3] = AddCar.CarWithThreeSeats("Vokswagen",  "WRA3263136", "2016", new Driver(ServiceMethods.findEmployeeByID(6)), new Seat(ServiceMethods.findEmployeeByID(10)), new Seat(ServiceMethods.findEmployeeByID(2)));;
 		table[4] = AddCar.CarWithThreeSeats("Vokswagen",  "WX7654333", "2014", new Driver(ServiceMethods.findEmployeeByID(7)), new Seat(ServiceMethods.findEmployeeByID(11)), new Seat(ServiceMethods.findEmployeeByID(3)));;
-		
+//		showCarObjectStorage();
 	}
+
+//	public void showCarObjectStorage() {
+//		for (Car c: table) {
+//			System.out.println("DDDDAAAAAA");
```
Comment (LukPrasek) [inline src/car/CarObjectsStorage.java:21]: check here

File: src/Main.java (hunk 1 above)
Comment (MatDls) [reply src/Main.java:74]: looks good

File: src/car/CarObjectsStorage.java (hunk 2 above)
Comment (MatDls) [reply src/car/CarObjectsStorage.java:21]: great job!

**Pull request merged**